airflow/utils/file.py

```diff
@@ -379,3 +382,9 @@ def iter_airflow_imports(file_path: str) -> Generator[str, None, None]:
     for m in _find_imported_modules(parsed):
         if m.startswith("airflow."):
             yield m
+
+
+def get_unique_dag_module_name(file_path):
```

Comment:
I am not sure if this is the right place to keep this utility function. Please do suggest where it is good to be placed.